msteams-platform/publishing/office-store-guidance.md

```diff
@@ -34,7 +34,7 @@ In the Seller Dashboard:
    * **Testing notes** – Please ensure you provide enough information for our validation team to successfully load and test your experience. This includes providing sign-in instructions and test accounts, as well as any other notes that might assist in the review of your product.
    * **My app calls, support, contains, or uses cryptography or encryption** – This checkbox helps to ensure that your product is not using cryptography in a way that would prohibit distribution through the Office Store. See [Export restrictions on cryptography](https://docs.microsoft.com/en-us/windows/uwp/security/export-restrictions-on-cryptography) for a detailed explanation.
    * **My app calls, support, contains, or uses cryptography or encryption** – This checkbox helps to ensure taht your product is not using cryptography in a way that would prohibit distribution through AppSource. See [Export restrictions on cryptography](https://docs.microsoft.com/en-us/windows/uwp/security/export-restrictions-on-cryptography) for a detailed explanation.
-   * **App logo** – Use your full-color 192&times;192 icon.
+   * **App logo** – Use the icon you want to appear in the AppSource catalog. The Teams gallery will use the icon from your manifest.
```

Comment:
Might be slightly more clear to say 

Use the icon you want to appear in the AppSource catalog. (Note that the gallery within the Teams app will use the icon from your manifest.)